api/src/main/java/jakarta/enterprise/invoke/Invokable.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 Red Hat and others
+ *
+ * This program and the accompanying materials are made available under the
+ * Apache Software License 2.0 which is available at:
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package jakarta.enterprise.invoke;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Marks a method as <em>invokable</em>. For each invokable method belonging to
+ * a managed bean class, the CDI container allows obtaining an {@link Invoker}
+ * through which the method can be invoked.
+ * <p>
+ * When this annotation is present on a class or interface, it marks all non-private
+ * methods (except constructors) declared in this class as invokable. Note that this
+ * does not apply to methods declared in superclasses or superinterfaces.
+ * <p>
+ * Attempting to mark a private method or a constructor of a managed bean class
+ * as invokable is a definition error.
+ * <p>
+ * Methods that are marked as invokable but do not belong to managed beans are ignored.
+ * <p>
+ * Note that multiple managed beans may inherit an invokable method from a common
+ * supertype. In that case, each bean conceptually has its own invokable method
+ * and for example an invoker obtained for one bean cannot be used to invoke
+ * the method on the other bean.
+ * <p>
+ * This annotation may be used as a meta-annotation. Classes and methods annotated with
+ * an annotation that is directly or transitively meta-annotated {@code @Invokable}
+ * are treated as if they were directly annotated {@code @Invokable}.
+ *
+ * @since 5.0
+ */
+@Target({ElementType.METHOD, ElementType.TYPE})
```

Comment:
needs to be a meta annotation

Reply:
`TYPE` implies `ANNOTATION_TYPE`, at least per `ElementType` javadoc, so this should be enough. But can do that explicitly, yea.

Reply:
The annotation is gone now.